assets/scss/base/_icons.scss

```diff
@@ -114,6 +114,29 @@ Styleguide Icons.Exclamation
 }
 
 /*
+Tick
+
+The tick icon is used to highlight actioned content to the user.
+
+Markup:
+<i class="tick"></i>
```

Comment:
In the example image attached to the pr the tick is there in order to show an item in the list has been actioned/completed. With that in mind people using screen readers would need to know this information